tests/t0116_gateway_cache_test.go

```diff
@@ -138,47 +138,47 @@ func TestGatewayCache(t *testing.T) {
 		{
 			Name: "GET for /ipfs/ file with matching Etag in If-None-Match returns 304 Not Modified",
 			Request: Request().
-				Path("ipfs/%s/root2/root3/root4/index.html", fixture.MustGetCid()).
+				Path("ipfs/{{}}/root2/root3/root4/index.html", fixture.MustGetCid()).
 				Headers(
-					Header("If-None-Match", fmt.Sprintf("\"%s\"", fixture.MustGetCid("root2", "root3", "root4", "index.html"))),
+					Header("If-None-Match", Templated("\"{{}}\"", fixture.MustGetCid("root2", "root3", "root4", "index.html"))),
```

Comment:
This still introduces some golang-specific escaping.
I wonder if we could avoid escaping quotes by backticks?

```suggestion
					Header("If-None-Match", Verbatim(`"{{etag-of-index-html}}"`, fixture.MustGetCid("root2", "root3", "root4", "index.html"))),
```

Reply:
Really cool, I switched every string with escaping to use backticks